packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,116 +2,174 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-$thumb-height: var(--iui-size-m); // 16px
-$track-height: var(--iui-size-2xs); // 4px
-$tick-height: var(--iui-size-s); // 12px
+// #region Slider container
+.iui-slider-container {
+  --_iui-slider-container-gap: var(--iui-size-s);
+  --_iui-slider-rail-color: var(--iui-color-border);
+  --_iui-slider-thumb-border-color: var(--iui-color-border-foreground);
+  --_iui-slider-thumb-color: var(--iui-color-background);
+  --_iui-slider-thumb-cursor: grab;
+  --_iui-slider-thumb-size: var(--iui-size-m);
+  --_iui-slider-tick-size: var(--iui-size-s);
+  --_iui-slider-tick-thickness: 1px;
+  --_iui-slider-track-color: var(--iui-color-border-accent);
+  --_iui-slider-track-thickness: calc(var(--_iui-slider-thumb-size) * 0.25);
+
+  display: grid;
+
+  &[data-iui-orientation='horizontal'] {
+    grid-template-columns: auto 1fr auto;
+    grid-template-areas: 'min content max';
```

Comment:
the grid areas make sense here but if i noticed you are using numbers in some places, particularly in pseudo elements. could you explain all parts of the grid?
<img width="318" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/e3d9e4d6-7bac-41d7-a43d-7b357b6112df">
<img width="211" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/a416cf43-75d8-4984-abc2-35d59d048af2">
<img width="419" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/82509da9-da0b-4df6-96b6-7161d00e4c52">
<img width="299" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/a2179e7b-a2f3-464f-8d60-5c72a92ca474">

Reply:
I typed out a much longer response to this, but it never saved. 😓. 

I have replaced the `grid-row: 1 / 3;` to just use `grid-row-start: 1;` for consistency.  Without using these, you get this: 
<img width="472" alt="Screenshot 2023-09-19 at 5 34 58 PM" src="https://github.com/iTwin/iTwinUI/assets/849817/9b4dc3f9-9998-4dd1-939c-6afdd9824eea">
<img width="85" alt="image" src="https://github.com/iTwin/iTwinUI/assets/849817/099c54f1-46ef-4221-bc32-5502eb955923">

Reply:
if you make `.iui-slider` a flexbox (see [comment](https://github.com/iTwin/iTwinUI/pull/1469#discussion_r1331921337)), then it should work automatically without needing explicit grid line numbers

Reply:
`.iui-slider` is now using `display: flex`. https://github.com/iTwin/iTwinUI/pull/1469#discussion_r1332116719